src/Language/PureScript/Docs/AsMarkdown.hs

```diff
@@ -62,6 +64,21 @@ codeToString = outputWith elemAsMarkdown
   elemAsMarkdown (Keyword x) = x
   elemAsMarkdown Space       = " "
 
+fixityAsMarkdown :: P.Fixity -> Docs
+fixityAsMarkdown (P.Fixity associativity precedence) = do
+  tell' $ concat [ "associativity: **"
```

Comment:
Come to think of it, "associativity: left-associative" is a bit redundant. I think it would be a bit nicer to just say "left-associative" (assuming we stick with the English style).